### ISSUE TITLE
As a user, I want to be able to read the docs so that I can learn how to install, configure, and use Adoptium products .

### ISSUE THREAD

**Issue by hendrikebbers** [AdoptOpenJDK/website-adoptium#17]

**Comment by fatihsolmaz22** [comment]
**Persona**
![Persona.png](https://user-images.githubusercontent.com/9443847/111908297-fb220d00-8a58-11eb-88c1-7e671fd369eb.png)

**Description**
As a user, I want an easily accessable and understandable docs on the website, which provides a useful instruction on how to install, configure and use Adoptium products.

**Acceptance criteria**
- A basic text-based instruction is shown in the browser
- User is capable of attaining crucial information through the docs
- Works on almost every browser / OS

**Assignees**
haltetim

**Comment by fatihsolmaz22** [comment]
Yes, preferably this would be an option to consider in the future. But I would focus on the initial creation and placement of the docs first.

**Comment by hiddenmind3** [comment]
Questions for PO:
Are the docs only accessible on the website or downloadable as a pdf?
What format are the docs saved in (markdown, pdf)?
Are the docs on a separate webpage, or are docs (such as instructions) linked on the download page?

**Comment by tresf** [comment]
> "Using Oracle today but it is no longer working"

This challenge does not seem accurate.  What would be more accurate is:

> "Using Oracle today, but concerns about the license"

I've never heard or seen an Oracle version of Java stop working.  I don't think it's a typical "challenge".

**Comment by hendrikebbers** [comment]
@hiddenmind3 
- The docs should be written like posts in markdown or asciidoc and html Contest should be created at build time based on the sources.
- We could create a PDF in the same way but that is currently not needed
- The HTML rendering of the docs should be integrated in the Adoptium page
- Some docs (installation for Java consumer) must be available in multiple languages.
